x-pack/plugins/security_solution/public/flyout/right/components/expand_detail_button.stories.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { Story } from '@storybook/react';
+import { ExpandableFlyoutContext } from '@kbn/expandable-flyout/src/context';
+import { ExpandDetailButton } from './expand_detail_button';
+import { RightPanelContext } from '../context';
+
+export default {
+  component: ExpandDetailButton,
+  title: 'Flyout/ExpandDetailButton',
+};
+
+export const Expand: Story<void> = () => {
```

Comment:
This is sweet. You should demo this during the next team wide demo 😄

Reply:
yeah I really like Storybook, it's nice when you review a PR to be able to look at a component and its different states. Happy to demo it!